qemu/tests/virtio_console.py

```diff
@@ -1805,6 +1807,14 @@ def test_failed_boot():
         :param cfg: virtio_console_params - Expected error message.
         """
         exp_error_message = params['virtio_console_params']
+        version_params = params["version_params"]
+        qemu_binary = utils_misc.get_qemu_binary(params)
+        output = str(process.run(qemu_binary + " --version"))
+        re_comp = re.compile(r'\d+.\d+.\d+-\d+..+')
+        output_list = re_comp.findall(output)
+        if not re.search(version_params, output_list[0]):
+            params["extra_params"] = params.get("extra_params_rhev")
+            exp_error_message = params.get("virtio_console_params_rhev")
```

Comment:
no default value if virtio_console_params_rhev is set? 
if so, you could use params["virtio_xxxx"]
also the line above